File: fix_predict_strings.py
import re
with open("lib/src/ui_predict_cards.dart", "r", encoding="utf-8") as f:
    text = f.read()

text = text.replace(r'\${', '${')
text = text.replace(r'\$', '$')

with open("lib/src/ui_predict_cards.dart", "w", encoding="utf-8") as f:
    f.write(text)
print("Fixed ui_predict_cards.dart backslashes")

File: fix_predict_strings.dart
import 'dart:io';

void main() async {
  final file = File('lib/src/ui_predict_cards.dart');
  var text = await file.readAsString();
  text = text.replaceAll(r'\${', r'${');
  text = text.replaceAll(r'\$', r'$');
  await file.writeAsString(text);
  print("Fixed ui_predict_cards.dart backslashes");
}
